aws/resource_aws_ecs_task_definition.go

```diff
@@ -230,6 +230,56 @@ func resourceAwsEcsTaskDefinition() *schema.Resource {
 				}, false),
 			},
 
+			"proxy_configuration": {
```

Comment:
This attribute should include a `d.Set()` during the `Read` function so the resource can perform drift detection. 👍

Reply:
i believe this is resolved via 098e3a6e4c6eb8244b8cec3ad79286c7679ea2db but i'm unable to see how to validate it.

while debugging i added some `log.PrintLn`s and saw the logs appear as i expected them to if i generated a crash.log, but i'm not familiar enough with how this is used internally to perform better validation.

Reply:
The easiest way to verify the `Read` function is correct for new attributes is adding an `ImportStateVerify: true` step in the acceptance test, which will force Terraform to evaluate all attributes for the resource are exactly the same after import as the previous step. When the `d.Set()` is missing from `Read`, the `ImportStateVerify` testing will fail. 😄 

I'll comment this below as well, but for this resource adding that testing is:

```go
{
	ResourceName:      resourceName,
	ImportState:       true,
	ImportStateIdFunc: testAccAWSEcsTaskDefinitionImportStateIdFunc(resourceName),
	ImportStateVerify: true,
},
```